package.json

```diff
@@ -4,10 +4,10 @@
   "description": "Starter app from the ground up",
   "repository": "https://github.com/jaebradley/edx-cookie-cutter",
   "scripts": {
-    "build": "NODE_ENV=production BABEL_ENV=production $(npm bin)/webpack --config=config/webpack.prod.config.js",
+    "build": "NODE_ENV=production BABEL_ENV=production webpack --config=config/webpack.prod.config.js",
     "lint": "eslint --ext .js --ext .jsx .",
     "prepublishOnly": "npm run build",
-    "start": "NODE_ENV=development BABEL_ENV=development $(npm bin)/webpack-dev-server --config=config/webpack.dev.config.js --hot --inline --progress",
+    "start:dev": "NODE_ENV=development BABEL_ENV=development webpack-dev-server --config=config/webpack.dev.config.js --hot --inline --progress",
```

Comment:
What's the reasoning behind this change? I think all of our JS apps currently use `start` not `start:dev`. If we want to use `start:dev`, that's okay, but I think we should make all of our apps use the same command so it's consistent across edX.